contrib/cirrus/setup.sh

```diff
@@ -8,14 +8,29 @@ set -e
 # expectation mismatch.
 source $(dirname $0)/lib.sh
 
-req_env_vars OS_RELEASE_ID OS_RELEASE_VER GOSRC IN_PODMAN_IMAGE
+req_env_vars OS_RELEASE_ID OS_RELEASE_VER GOSRC IN_PODMAN_IMAGE CIRRUS_CHANGE_TITLE CIRRUS_PR_DRAFT
 
 msg "Disabling git repository owner-check system-wide."
 # Newer versions of git bark if repo. files are unexpectedly owned.
 # This mainly affects rootless and containerized testing.  But
 # the testing environment is disposable, so we don't care.=
 git config --system --add safe.directory $GOSRC
 
+# Support optional/draft testing using latest/greatest
+# podman-next COPR packages.  This requires a draft PR
+# to ensure changes also pass CI w/o package updates.
+if [[ ! "$OS_RELEASE_ID" =~ "debian" ]] && \
```

Comment:
Negative checks are so hard to read; and, in this case, wrong: dnf is a fedora-only thing, not a "not-debian" thing. (I.e., if we ever switch back to ubuntu, or add some other trendy new distro, this will fail). Would you mind changing this to a positive `= "fedora"` check?

Reply:
Sure.